src/main/kotlin/de/danielmaile/mpp/packet/PacketListener.kt

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Why not use the priorities from bukkit?

Reply:
I wasn't planning to use the same as bukkit at first and then just copied it, so you're right. Will do that